core/ecschema-editing/src/test/Validation/SchemaComparisonSameNameVersion.test.ts

```diff
@@ -0,0 +1,325 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Property, Schema, SchemaContext, SchemaItem } from "@itwin/ecschema-metadata";
+import { AnyDiagnostic, ISchemaChanges, ISchemaCompareReporter, SchemaChanges, SchemaCompareCodes, SchemaComparer } from "../../ecschema-editing";
+import { expect } from "chai";
+
+class SchemaCompareReporter implements ISchemaCompareReporter {
+  public changes: SchemaChanges[] = [];
+  public report(schemaChanges: ISchemaChanges): void {
+    this.changes.push(schemaChanges as SchemaChanges);
+  }
+}
+
+function findDiagnostic(diagnostics: AnyDiagnostic[], code: string, fullNameA: string, _fullNameB?: string, _propertyType?: string) {
```

Comment:
why have _fullNameB and _propertyType if you never use them?

Reply:
There was no need for them for those diagnostics, removed.